### ISSUE TITLE
Degra_loss during training for customized datasets

### ISSUE THREAD

**Issue by Ljiong201108** [Algolzw/daclip-uir#34]
Hi! Firstly, great thanks for your outstanding and inspiring works!

When I tried to train daclip for my datasets, the Contrastive_loss decreased normally and converged to ~0.15. However, the Degra_loss decreased only in first 2 epochs and then maintained ~5.7 which is relative huge compared to Contrastive_loss.

Does this situation also occur by your training steps? I would be appreciated if you could also show the loss curves so that I could compare it with mine.

**Comment by Algolzw** [comment]
Hi! We also observed that the degra_loss converges worse. But we also found that the trained model can predict degradation types very well. Thus, I guess we don't need an accurate estimation of the degradation embedding (but different degradation embeddings should be distinguishable).

**Comment by Ljiong201108** [comment]
I see, thanks for the information!
